Remotion/Web/Development.WebTesting/Web.Development.WebTesting.csproj

```diff
@@ -22,11 +22,10 @@
     <ProjectReference Include="..\Contracts.DiagnosticMetadata\Web.Contracts.DiagnosticMetadata.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Fork.Coypu" Version="3.1.1.2" />
-    <PackageReference Include="Microsoft.Edge.SeleniumTools" Version="3.141.1" />
+    <PackageReference Include="Fork.Coypu" Version="3.2.0.1" />
     <PackageReference Include="log4net" Version="2.0.12" />
-    <PackageReference Include="Selenium.Support" Version="3.141.0" />
-    <PackageReference Include="Selenium.WebDriver" Version="3.141.0" />
+    <PackageReference Include="Selenium.Support" Version="4.5.0" />
```

Comment:
So we can manage this: please keep this conversation open until we have started v3.20 on the develop branch. Please check with @MichaelKetting at the start of November.